src/studio/src/designer/frontend/app-development/features/appPublish/pages/InfoCard.tsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import classNames from 'classnames';
-import classes from './AltinnInformationCardForChildren.module.css';
+import classes from './InfoCard.module.css';
+import Illustration from './illustration-help-2-circle.svg';
 
 export interface IAltinnInformationCardComponentProvidedProps {
   headerText: string;
-  imageSource: string;
   shadow: boolean;
 }
 
-export const AltinnInformationCardForChildren = (
+export const InfoCard = (
```

Comment:
watch out: you are loosing the "ForChildren" context here, not that I noticed a difference.

Reply:
Component is only used in one place... moved to that context to slim down that enormous `shared`-folder.